src/main/java/pl/com/bottega/ecommerce/sales/domain/productscatalog/ProductData.java

```diff
@@ -32,8 +32,11 @@
 		
 	private ProductType type;
 
-	
-	ProductData(Id productId, Money price, String name, ProductType type, 
+	public static ProductData create(Id productId, Money price, String name, ProductType type, Date snapshotDate){
```

Comment:
no jest to jakiś sposób ;). Ale wyłącznie dla potrzeb testów nie powinno się takich rzeczy robić. Lepiej jest zrobić sobie builder czy metodę fabrykującą w przestrzeni testów